README.md

```diff
@@ -1 +1,32 @@
-# appleStore
\ No newline at end of file
+# appleStore ([The Odin Project](https://web.archive.org/web/20140301004610/http://www.apple.com/))
+
+This is the first Solo project of the Main HTML/CSS curriculum at [Microverse](https:www.microverse.org/) - @microverseinc. 
+
+just a clone of an old archived ([apple](https://web.archive.org/web/20140301004610/http://www.apple.com/)) page: 
+
+![demopage](demopage.png)
+
+## Built With
+
+- HTML5 
+- CSS3
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/kelibst/appleStore/tree/featured-branch/index.html)
```

Comment:
The link is not working, it should be: https://raw.githack.com/kelibst/appleStore/featured-branch/index.html

Reply:
Thanks. I will make the necessary changes.